src/Connection.cpp

```diff
@@ -366,6 +367,23 @@ bool Connection::readBody()
 	return true;
 }
 
+bool Connection::readRequestHeadersAndBody()
```

Comment:
What is that?

Reply:
Was meant to declutter handleConnection, at the beggining when I took over the router, you don't like it ?

Reply:
You can do then directly also the easy fix that would make Daniil (and you) happy. So we solve the problem of the parsing core.

Reply:
I don't understand your comment, but I can reverse it if you want, I think I did it at the beginning of the branch and didn't realized it was your part (as our concept of core is a bit flawed)

Reply:
Let's leave it as it is, I will check it at a later point. Now I'm aware that it has been changed.